Packages/com.unity.inputsystem/InputSystem/Editor/ProjectWideActions/ProjectWideActionsAsset.cs

```diff
@@ -8,28 +8,29 @@
 
 namespace UnityEngine.InputSystem.Editor
 {
+    class ProjectWideActionsAssetPostprocessor : AssetPostprocessor
+    {
+        static void OnPostprocessAllAssets(string[] importedAssets, string[] deletedAssets, string[] movedAssets, string[] movedFromAssetPaths, bool didDomainReload)
+        {
+            if (ArrayHelpers.Contains(importedAssets, ProjectWideActionsAsset.kAssetPath))
+            {
+                AssetDatabase.DisallowAutoRefresh();
+                ProjectWideActionsAsset.CreateRoslynAdditionalFile();
+                AssetDatabase.AllowAutoRefresh();
+            }
+        }
+    }
+
     internal static class ProjectWideActionsAsset
     {
-        internal const string kDefaultAssetPath = "Packages/com.unity.inputsystem/InputSystem/Editor/ProjectWideActions/ProjectWideActionsTemplate.json";
+        internal const string kTemplateAssetPath = "Packages/com.unity.inputsystem/InputSystem/Editor/ProjectWideActions/ProjectWideActionsTemplate.json";
```

Comment:
None of the changes of this class relates to Type-safe API right? (Could have been a separate PR against develop)